Appraisals

```diff
@@ -0,0 +1,20 @@
+appraise "rails-7" do
+  gem "actionpack", "~> 7.1.0"
+  gem "activesupport", "~> 7.1.0"
+  gem "activerecord", "~> 7.1.0"
+  gem "sqlite3", "~> 1.7.0"
+end
+
+appraise "rails-8" do
+  gem "actionpack", "~> 8.0.0"
+  gem "activesupport", "~> 8.0.0"
+  gem "activerecord", "~> 8.0.0"
+  gem "sqlite3", "~> 2.4.0"
+end
+
+appraise "rails-edge" do
+  gem "actionpack", git: "https://github.com/rails/rails.git", branch: "main"
+  gem "activesupport", git: "https://github.com/rails/rails.git", branch: "main"
+  gem "activerecord", git: "https://github.com/rails/rails.git", branch: "main"
+  gem "sqlite3", "~> 2.4.0"
+end
```

Comment:
I stopped using `Appraisals` because it felt it was not doing a lot more than setting `BUNDLE_GEMFILE` for each `Gemfile` you can to test against and add some tasks to do some looping over gemfiles. But again, you did all this work to bring things up to date which is super appreciated so I'm happy to go back to `Appraisals` if you find it better.